src/axom/inlet/Table.hpp

```diff
@@ -476,6 +685,20 @@ class Table : public SchemaCreator, public std::enable_shared_from_this<Table>
    */
   bool hasField(const std::string& fieldName);
 
+  /*!
+   *****************************************************************************
+   * \brief Return whether a Table or Field with the given name is present in 
+   * this Table's subtree.
+   *
+   * \return Boolean value indicating whether this Table's subtree contains a
+   * Field or Table with the given name.
+   *****************************************************************************
+   */
+  bool contains(const std::string& name)
+  {
+    return hasTable(name) || hasField(name);
```

Comment:
Can `hasTable`, `hasField`, and `contains` be collapsed into just `contains`?  I don't think it makes sense to have a field and a table of the same name at the same scope.

Reply:
Not sure... the user might need to have different logic for when something is a `Table` vs. when something is a `Field`.